src/test/java/com/aws/greengrass/logmanager/util/TestUtils.java

```diff
@@ -35,4 +43,23 @@ public static LogFile createLogFileWithSize(URI uri, int bytesNeeded) throws IOE
         writeFile(file, bytesArray);
         return file;
     }
+
```

Comment:
We have this in a bunch of other codebases. Eventually we should move it to the nucleus

Reply:
Can't we use the eventually matcher? https://github.com/grantwest/eventually-matchers

Reply:
Did not know that was a thing. 😂 I will check it out